rir/src/compiler/translations/pir_2_rir.cpp

```diff
@@ -929,11 +990,36 @@ size_t Pir2Rir::compile(Context& ctx, Code* code) {
                 break;
             }
             case Tag::StaticCall: {
-                assert(false && "not yet implemented.");
+                auto call = StaticCall::Cast(instr);
+                setEnv(it, call->env());
+
+                compiler.compile(call->cls(), call->origin());
+                // push callee - this stores it into constant pool - discuss!
+                cs << BC::push(call->origin());
+
+                std::vector<FunIdxT> callArgs;
+                call->eachCallArg([&](Value* arg) {
+                    // TODO: for now, ignore the eager value in MkArg
```

Comment:
actually if all of them have a non-missing eager arg, then you could do callStack instead...